harness/benchmarks/typed_gd/Simple.gd

```diff
@@ -0,0 +1,51 @@
+func benchmark_simple_add() -> int:
+	var a: int  = 1
+	var b: int = 2
+	return a + b
+
+func benchmark_avg() -> int:
+	var size: int = 10000
+	var total: int = 0
+	for i in range(size):
+		total += i
+	return total / size
+
+func benchmark_vectors() -> Vector3:
+	var b: Transform = Transform()
+	b = b.rotated(Vector3.UP, deg2rad(60))
+	b = b.scaled(Vector3(0.5, 0.5, 0.5))
+
+	var s: Vector3 = Vector3()
+	for i in range(1000):
+		var v: Vector3 = Vector3(i, i, i)
+		v = b.xform(v)
+		s = s + v
+
+	return s
+
+func benchmark_vectors2_only() -> Vector2:
+	var s: Vector2 = Vector2()
+	for i in range(1000):
+		var v = Vector2(i, i)
```

Comment:
@CedNaru Not typed. Either type it explicitly or use `:=`

Reply:
done